src/javaharness/java/arcs/crdt/CollectionOperation.java

```diff
@@ -23,8 +23,6 @@ public CollectionOperation(Type type, T t, VersionMap clock, String actor) {
       case REMOVE:
         this.removed = Optional.of(t);
         break;
-      default:
```

Comment:
Maybe we should at least log something? Was the error about throwing inside of a ctor, or around use of `default`?

Reply:
According to error prone, all switch enums are handled, so the fallthrough is impossible